### PR TITLE
Floating news added #41

### PR DESCRIPTION
**Pull request by Ahrazshamim**
A floating news section was added.

### CODE REVIEW

**Review by himanshuvarandani — APPROVED** [review]

File: pages/index.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ export default function Home() {
   return (
     <div className={styles.container}>
       <Header />
+      <div className="bg-gray-200 text-blue-900 w-full mt-8 lg:mr-2 lg:p-2 p-1"  >
+  <marquee>
```
Comment (pnijhara) [inline pages/index.js:14]: Give proper indentation.

File: pages/index.js [hunk 2]
```diff
@@ -10,6 +10,11 @@ export default function Home() {
...
+    <p> SSGSA results (2021-2022) annonced </p>
```
Comment (pnijhara) [inline pages/index.js:15]: This should be "announced".

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 2 above)
Comment (Ahrazshamim) [reply pages/index.js]: updated, please review

**Review by pnijhara — APPROVED** [review]
This looks good. Thank you for the PR.

**Pull request merged**